python/pipeline/preprocess.py

```diff
@@ -779,24 +815,31 @@ def update_img(i):
                                         interval=1000 / fps)
 
         # Save animation
+        if not filename.endswith('.mp4'):
+            filename += '.mp4'
         print('Saving video at:', filename)
         print('If this takes too long, stop it and call again with dpi < 200 (default)')
         video.save(filename, dpi=dpi)
 
         return fig
 
-    def plot_contours(self, slice=1, channel=1):
-        """ Draw contours of all masks over the correlation image.
+    def plot_contours(self, slice=1, channel=1, first_n=None):
+        """ Draw contours of masks over the correlation image.
 
         :param slice: Scan slice to use
         :param channel: Scan channel to use
+        :param first_n: Number of masks to plot. None for all.
```

Comment:
Set None as default. I think is better than 0 which means something, it could be -1 but I don't see why it would be better